tests/phpunit.xml

```diff
@@ -3,7 +3,7 @@
 	convertErrorsToExceptions="true" convertNoticesToExceptions="false"
 	convertWarningsToExceptions="false">
 	<testsuite name="UnitTests">
-		<directory>./unit</directory>
+		<directory>./Unit</directory>
```

Comment:
The directory name starts with a lowercase u, this change is not needed.

Reply:
Fixed